portfolio/src/main/webapp/js/script.js

```diff
@@ -49,6 +52,34 @@ function makeChangeDescriptionCallBack(projectIndex) {
     }
 }
 
+/**
+ * Draws charts for words frequencies.
+ */
+function drawCharts() {
+    fetch('/words', {method: 'GET'}).then(response => response.json()).then((wordsArray) => {
+        const chartsDOM = document.querySelector('.charts-divs');
+        var index = 0;
+        wordsArray.forEach(album => {
+            var data = new google.visualization.DataTable();
+            data.addColumn('string', 'Words');
+            data.addColumn('number', 'Number of appearances');
+            data.addRows(Object.entries(album.words).map(([_, wordObj]) => [wordObj.word, wordObj.frequency]));
```

Comment:
nit: the line still seems longer than 80 characters, which is the limit recommended in the style guide... can you please break it?